flight/Modules/CharacterOSD/charonscreendisplay.c

```diff
@@ -237,7 +238,7 @@ static void update_telemetry(charosd_state_t state)
 	SystemStatsFlightTimeGet(&state->telemetry.system.flight_time);
 	AttitudeActualGet(&state->telemetry.attitude_actual);
 	PositionActualGet(&state->telemetry.position_actual);
-	ManualControlCommandThrottleGet(&state->telemetry.manual.throttle);
+	StabilizationDesiredThrustGet(&state->telemetry.manual.thrust);
```

Comment:
OSDs are moved to using the "thrust" value which is far more useful (and can represent negative numbers across aircraft types).